### PR TITLE
Added interfaces for CRUD operations.

### PR DESCRIPTION
**Pull request by SergeyPrudnikov-dev**

### CODE REVIEW

File: src/main/java/com/epam/university/commission/repository/ICrudRepository.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @param <ID> - the type of the id of the entity the ICrudRepository manages.
+ * @param <T>  - the domain type the CrudRepository manages.
+ * <p/>
+ * University-commission 2021  epam.com
+ * <p/>
+ * Date: 04/20/2021
+ *
+ * @author Siarhei Prudnikau1
+ */
+public interface ICrudRepository<T, ID extends Serializable>{
```
Comment (andrei-kuzniatsou) [inline src/main/java/com/epam/university/commission/repository/ICrudRepository.java:16]: <T, ID extends Serializable> what for do we need this?

File: src/main/java/com/epam/university/commission/repository/ICrudRepository.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
```
Comment (andrei-kuzniatsou) [inline src/main/java/com/epam/university/commission/repository/ICrudRepository.java:17]: add crud methods

File: src/main/java/com/epam/university/commission/repository/ICrudRepository.java (hunk 1 above)
Comment (SergeyPrudnikov-dev) [reply src/main/java/com/epam/university/commission/repository/ICrudRepository.java:16]: It is mistake.

**Comment by andrei-kuzniatsou** [comment]
no objections

**Pull request merged**